3rdparty/bazel/org_golang/go_tls_max_handshake_size.patch

```diff
@@ -1,11 +1,11 @@
 --- src/crypto/tls/common.go
 +++ src/crypto/tls/common.go
-@@ -62,7 +62,7 @@
- 	maxCiphertext      = 16384 + 2048 // maximum ciphertext payload length
- 	maxCiphertextTLS13 = 16384 + 256  // maximum ciphertext length in TLS 1.3
- 	recordHeaderLen    = 5            // record header length
--	maxHandshake       = 65536        // maximum handshake we support (protocol max is 16 MB)
-+	maxHandshake       = 262144       // maximum handshake we support (protocol max is 16 MB)
- 	maxUselessRecords  = 16           // maximum number of consecutive non-advancing records
+@@ -64,7 +64,7 @@ const (
+ 	maxCiphertext              = 16384 + 2048 // maximum ciphertext payload length
+ 	maxCiphertextTLS13         = 16384 + 256  // maximum ciphertext length in TLS 1.3
+ 	recordHeaderLen            = 5            // record header length
+-	maxHandshake               = 65536        // maximum handshake we support (protocol max is 16 MB)
++	maxHandshake               = 262144        // maximum handshake we support (protocol max is 16 MB)
+ 	maxHandshakeCertificateMsg = 262144       // maximum certificate message size (256 KiB)
```

Comment:
Do we even need our patch with 1.23.2?
This seems like it accomplishes what we did with out patch.

Reply:
https://github.com/golang/go/issues/50773
We can probably drop this patch now, but lets move that to another PR